src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.logo,
+.nav {
+  display: block;
+}
+
+.nav {
+  align-content: center;
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family 'Roboto' is used here, but make sure it is correctly imported in your HTML file. It seems to be imported via a link tag, which is good practice.